### PR TITLE
Numbers task

### PR DESCRIPTION
**Pull request by andreamamchur**

### CODE REVIEW

File: Exercises/numbers/numbers.js [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+const variableDecimal = 123;
+
+console.log(variableDecimal.toString(2));
+console.log(variableDecimal.toString(8));
+console.log(variableDecimal.toString(16));
+
+// use `Math.random` to generate random value from `0` to `100`.
+
+function random(min, max) {
+  return (Math.round(min - 0.5 + Math.random() * (max - min + 1)));
```
Comment (Mike-Savin) [inline Exercises/numbers/numbers.js:26]: You can skip the most outer brackets

File: Exercises/variables-declaration/variables-declaration.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Declare a variable which will be encapsulated from global scope.
+
+const variableGlobal = 'global'; //global
```
Comment (Mike-Savin) [inline Exercises/variables-declaration/variables-declaration.js:3]: Each time when you create a new branch for a new task you should create it not from other task branch but from `master`. Then in you PR you will have only changes that are related to your current task.
Currently you created branch `numbers` from `variables-declaration` branch and that's why you have unnecessary changes in this branch.
You can leave this PR for now but in future please base your new branches on master so we can check your code only in files that should be checked.

**Review by Mike-Savin — APPROVED** [review]
Looks good. I approved it but please see my comments

File: Exercises/variables-declaration/variables-declaration.js (hunk 2 above)
Comment (andreamamchur) [reply Exercises/variables-declaration/variables-declaration.js:3]: Mike, thank you for review and comment. Let me clarify the steps to skip future inconvenience
1. switch to the master from the current branch
2. create new branch 
3. make commit from the created branch

**Pull request merged**